corehq/apps/hqwebapp/static/hqwebapp/less/style-imports.less

```diff
@@ -32,3 +32,4 @@
 @import "_hq/typography.less";
 @import "_hq/svg.less";
 @import "_hq/utils.less";
+@import "_hq/optimize.less";
```

Comment:
Super-nitpick: putting this in the list alphabetically makes it more obvious that the order of these imports isn't important.